.gitignore

```diff
@@ -35,3 +35,6 @@ target
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 
 hs_err_pid*
+
+
+*/alf_data
```

Comment:
Normally the alf_data directory is not seen. Is this really needed for everyone? Looks like it may be because of local config.